src/main/java/org/example/homework5/EvenDigitSum.java

```diff
@@ -0,0 +1,27 @@
+package org.example.homework5;
+
+import java.lang.reflect.Array;
+
+public class EvenDigitSum {
+    public static void main(String[] args) {
+        int initialNumber = 2;
+        System.out.println(getEvenDigitSum(initialNumber));
+    }
+    public static int getEvenDigitSum(int number) {
+        if (number < 0) {
+            return -1;
+        }
+
+        int result = 0;
+        char[] stringNumber = Integer.toString(number).toCharArray();
```

Comment:
Technically your solution is correct, but not efficient.
You can take a look at this one:
```
        int result = 0;
        while (number > 0) {
            int digit = number % 10;
            if (digit % 2 == 0) {
                sum += digit;
            }
            number /= 10;
        }
        return result;
```
It does not require an additional array and int to string manipulations

Reply:
Fixed, but left old solution as commented.